### PR TITLE
Adds instance to allow users to initialize Taplytics themselves.

### PR DESCRIPTION
**Pull request by ladanazita**

### CODE REVIEW

File: Segment-Taplytics/Classes/SEGTaplyticsIntegration.h [hunk 1]
```diff
@@ -20,4 +20,6 @@
 
 - (instancetype)initWithSettings:(NSDictionary *)settings andTaplytics:(id)taplyticsClass;
 
+- (instancetype)initWithoutSettings: andTaplytics:(id)taplyticsClass;
```
Comment (f2prateek) [inline Segment-Taplytics/Classes/SEGTaplyticsIntegration.h]: Surprised this synatx works but how about:

`- (instancetype)initWithTaplytics:(id)taplyticsClass;`

File: Segment-Taplytics/Classes/SEGTaplyticsIntegrationFactory.m [hunk 2]
```diff
@@ -22,6 +21,12 @@ + (instancetype)instance
     return sharedInstance;
 }
 
++ (instancetype)initializeTaplyticsManually
+{
+    static SEGTaplyticsIntegrationFactory *sharedInstance;
```
Comment (f2prateek) [inline Segment-Taplytics/Classes/SEGTaplyticsIntegrationFactory.m]: Think about what this is doing.

You aren't creating a factory here. You defined a variable, which is nil and returned it/nil.

You still need to creat a facttory that calls your `initWithoutSettings` initializer.

File: Segment-Taplytics/Classes/SEGTaplyticsIntegrationFactory.m [hunk 3]
```diff
@@ -33,6 +38,13 @@ - (instancetype)init
     return [[SEGTaplyticsIntegration alloc] initWithSettings:settings];
 }
 
+- (id<SEGIntegration>)createWithoutSettings
+{
+    return [[SEGTaplyticsIntegration alloc] init];
```
Comment (f2prateek) [inline Segment-Taplytics/Classes/SEGTaplyticsIntegrationFactory.m]: where is the `init` method defined? your new initializer was called `initWithoutSettings: andTaplytics:(id)taplyticsClass;`.

File: Segment-Taplytics/Classes/SEGTaplyticsIntegration.h (hunk 1 above)
Comment (f2prateek) [inline Segment-Taplytics/Classes/SEGTaplyticsIntegration.h]: This isn't being called by your factory. SO this ends up being unused.

**Review by tonyxiao — APPROVED** [review]

**Pull request merged**